lib/engine/config/game/g_1882.rb

```diff
@@ -263,6 +263,12 @@ module G1882
           "hexes": [
             "H4"
           ]
+        },
+        {
+          "type": "exchange",
```

Comment:
this is a very specific ability, i don't know if we should put it in the generic framework

Reply:
i'm struggling to find anything similar.

Is there a suggestion on how to do it which won't look terrible :)